README.md

```diff
@@ -95,6 +95,6 @@ repeat these steps to upgrade to future releases.
 ## Building
 
 * [Linux](docs/build-instructions/linux.md)
-* [OS X](docs/build-instructions/os-x.md)
+* [macOS](docs/build-instructions/macos.md)
```

Comment:
Can't you just symlink the old name to the new name to ensure links don't break?